docs/mssautoplot.rst

```diff
@@ -0,0 +1,81 @@
+ 
+How to use
+----------
+
+The CLI tool has the following parameters:
+
++--------------+-------+----------------------------------------------------------------------+
+| Parameter    | Type  |         Description                                                  |
++--------------+-------+----------------------------------------------------------------------+
+| ``--cpath``  | TEXT  |   Path of the config file where the configuration is specified.      |
++--------------+-------+----------------------------------------------------------------------+
+| ``--ftrack`` | TEXT  |   Flight track.                                                      |
++--------------+-------+----------------------------------------------------------------------+
+| ``--itime``  | TEXT  |   Initial time.                                                      |   
++--------------+-------+----------------------------------------------------------------------+
+| ``--vtime``  | TEXT  |   Valid time.                                                        |   
++--------------+-------+----------------------------------------------------------------------+
+| ``--intv``   |INTEGER|   Time interval in hours.                                            |  
++--------------+-------+----------------------------------------------------------------------+
+| ``--stime``  | TEXT  |   Starting time for downloading multiple plots with a fixed interval.|
++--------------+-------+----------------------------------------------------------------------+
+| ``--etime``  | TEXT  |   Ending time for downloading multiple plots with a fixed interval.  |
++--------------+-------+----------------------------------------------------------------------+
+
+A short description of how to start the program is given by the ``--help`` option.
+
+Examples 
+~~~~~~~~
+
+Here are a few examples on how to use this tool,
+
+1. ``mss autoplot --cpath mss_autoplot.json``
+
+The above command downloads the required number of plots with the default settings from mss_autoplot.json into the output folder.
+
+2. ``mss autoplot --cpath mss_autoplot.json --itime="" --vtime="2019-09-02T00:00:00"``
+
+The above command will download all the plots configured in mss_autoplot.json of initial time "" and valid time "2019-09-02T00:00:00". This can be used to create a daily "standard set" of plots with/without an actual flight track, e.g., for daily morning briefings
+
+For downloading plots of multiple flight tracks, specify the flight track and it's configuration in mss_autoplot.json. This can be used to create a "standard set" of plots for all actual flights of a campaign. See below:
+
+"automated_plotting_flights": [
+           ["flight1", "section1", "vertical1", "filename1", "init_time1", "time1"]
+           ["flight2", "section2, "vertical2", "filename2", "init_time2", "time2"]]
+
+3. ``mss autoplot --cpath mss_autoplot.json --stime="2019-09-01T00:00:00" --etime="2019-09-02T00:00:00" --intv=6``
```

Comment:
the command is likly mssautoplot, one word.

in a future step it may become integrated into https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/mss.py